bin/migrate-oats-data/common/oats_to_alcs_planning_review_type.py

```diff
@@ -15,3 +18,10 @@ class OatsToAlcsPlanningReviewType(Enum):
     UTIL = "UEPP"
     AAP = "AAPP"
     APC = "MISC"
+
+
+class OatsToAlcsDecisionOutcomes(Enum):
+    END = AlcsPlanningReviewOutcomes.ENDO
```

Comment:
To avoid double value, whenever you are referencing enum value, you could use value here. END = AlcsPlanningReviewOutcomes.ENDO.value so later in code it will be just regular AlcsPlanningReviewOutcomes.END.value